xlint

```diff
@@ -60,8 +60,6 @@ variables_order() {
 			version=*) curr_index=3;;
 			revision=*) curr_index=4;;
 			archs=*) curr_index=5;;
-			wrksrc=*) curr_index=7;;
-			create_wrksrc=*) curr_index=8;;
```

Comment:
I'm going to restore `create_wrksrc`. Some templates explicitly add `create_wrksrc` to move the file hierarchy down,
and cp the source code into multiple directory for multiple build (with different build option). See `lua54-cjson`.

I think that's a good use case.